content/proposals.md

```diff
@@ -79,6 +79,19 @@ Please [see
 MSC1779](https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/1779-open-governance.md)
 for full details of the project's Guiding Principles.
 
+For the purposes of transparency and to underpin the "greater benefit"
+principle, proposals MUST disclose possible conflicts of interest where
+they might reasonably exist. Authors MAY do this by filling out the
+following checklist and publishing it together with their proposal.
```

Comment:
It is debatable whether RFC2119 language makes sense here. I found it helpful for differentiating between hard and optional requirements. I might be partial to this language by having worked with it in the spec for too long, however.